addons/website_forum/views/website_forum_profile.xml

```diff
@@ -56,7 +56,22 @@
             <t t-if="forum">
                 <div role="tabpanel" class="o_wforum_profile_tab tab-pane" id="questions">
                     <div class="mb-4">
-                        <h5 class="border-bottom pb-1">
+                        <h5 class="border-bottom pb-1 container">
+                            <t t-if="count_questions > 0">
+                                <t class="float-right" t-call="website.website_search_box_input">
+                                    <t t-set="_form_classes" t-valuef="form-inline float-right"/>
+                                    <t t-set="search_type" t-valuef="forums"/>
+                                    <t t-if="forum_id" t-set="action" t-value="'/forum/%s%s' % (forum_id, tag and ('/tag/%s/questions' % slug(tag)) or '')"/>
+                                    <t t-else="" t-set="action" t-value="'/forum/all%s' % (tag and ('/tag/%s/questions' % slug(tag)) or '')"/>
+                                    <t t-set="display_description" t-valuef="true"/>
+                                    <t t-set="display_detail" t-valuef="true"/>
+                                    <t t-set="placeholder" t-value="'Search Questions...'"/>
+                                    <input type="hidden" name="author" t-att-value="user.id"/>
+                                    <input t-if="filters" type="hidden" name="filters" t-att-value="filters"/>
+                                    <input t-if="request.env.user == user" type="hidden" name="my" t-att-value="mine"/>
```

Comment:
it look like this condition is used at many place, maybe we can give a variable `my_profile` to the qweb template, what do you think ? (and we will also remove `request` from the templates which is a good thing)